disneyland.proto

```diff
@@ -29,25 +29,19 @@ message RequestWithId {
     uint64 id = 1;
 }
 
-message ListJobStatusResponse {
-    repeated Job.Status status = 1;
-}
-
 message ListJobsRequest {
     uint32 how_many = 1;
     string project = 2;
     string kind = 3;
 }
 
-
 service Disneyland {
     rpc CreateJob (Job) returns (Job) {}
     rpc GetJob (RequestWithId) returns (Job) {}
     rpc ListJobs (ListJobsRequest) returns (ListOfJobs) {}
-
+    rpc CreateMultipeJobs (ListOfJobs) returns (ListOfJobs) {}
     rpc ModifyJob (Job) returns (Job) {} /** Merges passed Status, MetricValue and Metadata to db */
-
     rpc PullPendingJobs (ListJobsRequest) returns (ListOfJobs) {}
     rpc DeleteJob(RequestWithId) returns (Job) {}
-    //rpc BidiJobs (stream ListOfJobs) returns (stream ListJobStatusResponse) { }
+    rpc BidiJobs(stream ListJobsRequest) returns(stream Job) {}
```

Comment:
Bidirectional rpc should be expanded.

Server should be able to receive

- Information about worker — cpu/memory available/how many computational slots available
- Information about running jobs — cpu/memory usage, logs(if requested)

Client should be able to receive:

- New jobs
- Job cancelation signal

Also both client and server should be able to ping/pong each other.

Having all this information exchanges enables server to execute some smart scheduling decisions.